src/js/modules/infragistics.ui.editors.js

```diff
@@ -8132,7 +8184,23 @@
 			this._spin(this.options.spinDelta, true);
 		},
 		_handleSpinDownEvent: function () { // DateEditor
-			this._spin(-this.options.spinDelta, true);
+			this._spin(this._setNegativeDelta(this.options.spinDelta), true);
+		},
+		_setNegativeDelta: function (delta) {

```

Comment:
😦 Not super fond of this - the delta shouldn't need to be passed like that down the chain like that, especially with the new `_getDelta`. 
I'm okay with setting it aside till we have the time to redactor though.